main.go

```diff
@@ -0,0 +1,239 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"strconv"
+	"strings"
+
+	"encoding/json"
+
+	"github.com/Shopify/sarama"
+	"github.com/companieshouse/chs.go/avro"
+	"github.com/companieshouse/chs.go/avro/schema"
+	"github.com/companieshouse/chs.go/kafka/producer"
+	"github.com/companieshouse/kafka-management-tool/schemas"
+)
+
+// Assigns all flags to variables
+var (
+	brokerPtr         = flag.String("broker", "", "Broker address")
+	topicPtr          = flag.String("topic", "", "Topic name")
+	schemaPtr         = flag.String("schema", "", "Schema name")
+	schemaRegistryPtr = flag.String("schema-registry", "", "Schema Registry")
+	partitionPtr      = flag.Int64("partition", 0, "Partition (default: 0)")
+	offsetPtr         = flag.String("offset", "", "Offset number")
```

Comment:
some explanation of the format allowed for this field should be added i.e. the fact that it can be a single integer or a range with `-` used as a separator

Reply:
Good point, changed under [7efa1c2](https://github.com/companieshouse/kafka-management-tool/pull/1/commits/7efa1c27d45f02043561f0755d11caa68de5f81c)